src/test/java/com/gooddata/auditevent/AuditEventServiceAT.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2007-2017, GoodData(R) Corporation. All rights reserved.
+ */
+package com.gooddata.auditevent;
+
+import com.gooddata.AbstractGoodDataAT;
+import com.gooddata.collections.PageableList;
+import org.testng.annotations.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+
+public class AuditEventServiceAT extends AbstractGoodDataAT {
+
+    @Test
+    public void shouldListEventsForCurrentUser() throws Exception {
```

Comment:
Can we add an AT utilizing the domain resource?

Reply:
not easily at this time. first of all it requires domain admin account and we currently still don't run such tests (isolated_domain test group saga).
secondly there is currently no way how to obtain domain of current account, so we would need to introduce a new test parameter :(

Reply:
Hm that's not nice... Is there a plan to eventually add such tests? And if so, wouldn't it be worth it to add the test now and @Ignore it so that it's not forgotten?

Reply:
added